.gitignore

```
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```

we shouldn't ignore the lock file in the root folder, only in `examples/`

I think this should do:

```suggestion
/examples/**/package-lock.json
```